README.md

```diff
@@ -473,6 +473,48 @@ GNN support is rapidly evolving, please contact the team directly or on Slack fo
     
 See `help(g.search_graph)` for options
 
+### Knowledge Graph Embeddings
```

Comment:
* should say Link Prediction
* ... and anomaly detection for scoring
* linked to some explainers for rgcn: our demo notebooks + some paper

Reply:
Done, will add demo too